plugin/src/android/src/main/java/com/mparticle/cordova/MParticleCordovaPlugin.java

```diff
@@ -86,6 +86,14 @@ public void logEvent(final JSONArray args) throws JSONException {
         MParticle.getInstance().logEvent(event);
     }
 
+    public void logMPEvent(final JSONArray args) throws JSONException {
```

Comment:
I see that this is a newly added function. How were events logged previously? In the case of Xamarin for example, I tried to modify the original method rather than making a new one (in the Xamarin case, the method passed arguments rather than an event object, so I added an optional `shouldUploadEvent` arg to the existing function.

Not saying that's the way to go here, but trying to understand your approach better.

Reply:
@einsteinx2 So if you checkout 'logEvent' above this you'll see that Cordova could only log simple events. It just passed in 'name', 'eventType', and 'customAttributes'. Since that lacked so much functionality but mirrored an existing method on the core sdks I thought added a full logEvent would be best and would avoid this being a breaking change for existing Cordova users.